src/main/java/org/team199/robot2022/subsystems/Climber.java

```diff
@@ -86,147 +103,86 @@ public void periodic() {
+      if (motor<1){
         left.set(0);
+        SmartDashboard.putString("Left Climber State", "Stopped");
+      }
+      if (motor>-1){
         right.set(0);
-        SmartDashboard.putString("Left climber is", "Stopped");
-        SmartDashboard.putString("Right climber is", "Stopped");
-    }
-
-    public void stopLeft() {
-        left.set(0);
-        SmartDashboard.putString("Left climber is", "Stopped");
-    }
-
-    public void stopRight() {
-        right.set(0);
-        SmartDashboard.putString("Right climber is", "Stopped");
-    }
-
-    public void stop(boolean interrupted) {
-        stop();
-    }
-
-    public void stopLeft(boolean interrupted) {
-        stopLeft();
-    }
-
-    public void stopRight(boolean interrupted) {
-        stopRight();
-    }
-
-    public double getLeftPosition() {
-        return leftEncoder.getPosition();
+        SmartDashboard.putString("Right Climber State", "Stopped");
+      }
     }
 
-    public double getRightPosition() {
-        return rightEncoder.getPosition();
+    public boolean isMotorExtended(int motor){//is the motor(s) extended?
+      if (motor<1 && leftEncoder.getPosition() < EncoderPos.extendLeft.value){
+        return false;
+      }
+      if (motor>-1 && rightEncoder.getPosition() < EncoderPos.extendRight.value){
+        return false;
+      }
+      return true;
     }
 
-    public boolean isLeftExtended() {
-        return getLeftPosition() >= extendPositionLeft;
+    public boolean isMotorRetracted(int motor){//is the motor(s) retracted?
+      if (motor<1 && leftEncoder.getPosition() > EncoderPos.retractLeft.value){
+        return false;
+      }
+      if (motor>-1 && rightEncoder.getPosition() > EncoderPos.retractRight.value){
+        return false;
+      }
+      return true;
     }
 
-    public boolean isRightExtended() {
-        return getRightPosition() >= extendPositionRight;
-    }
+    public static enum MotorSpeed{
+      retract(-((kDesiredRetractSpeedInps / kInPerSec) + (kVoltsToCounterTorque / 12))),// ~ -0.06151
+      extend(kDesiredExtendSpeedInps / kInPerSec),// ~0.30261
+      slowRetract(-((kSlowDesiredRetractSpeedInps / kInPerSec) + (kSlowVoltsToCounterTorque / 12))),// ~ -0.06151
+      slowExtend(kSlowDesiredExtendSpeedInps / kInPerSec);// ~0.30261
 
-    public boolean isLeftRetracted() {
-        return getLeftPosition() <= retractPositionLeft;
-    }
+      public final double value;
 
-    public boolean isRightRetracted() {
-        return getRightPosition() <= retractPositionRight;
-    }
+      private MotorSpeed(double value) {
+          this.value = value;
+      }
+    };
 
-    public boolean isRightResetExtended() {
-        return getRightPosition() >= 0;
-    }
 
-    public boolean isLeftResetExtended() {
-        return getLeftPosition() >= 0;
-    }
+    public static enum EncoderPos{
+      extendLeft(-5.317),
+      extendRight(5.315),
+      retractLeft(-1.151),
+      retractRight(-1.172),
```

Comment:
```suggestion
      extendLeft(6.315),
      extendRight(6.315),
      retractLeft(-0.6),
      retractRight(-0.6),
```
These values should match the previous encoder constants